### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by dy** [colorjs/contributing#3]
Logo can be enhanced in a way. Mb outline?

**Comment by dy** [comment]
![logo2](https://cloud.githubusercontent.com/assets/300067/23583537/603bc9fa-0115-11e7-80e4-4566604a424b.png)

**Comment by dy** [comment]
![logo3](https://cloud.githubusercontent.com/assets/300067/23583548/a031cae6-0115-11e7-96d3-4b9ed6681231.png)

**Comment by dy** [comment]
Anyways here is [project on figma](https://www.figma.com/file/eKiGfjQzbJVz9DnSzwwkpBU1/Logo?node-id=1%3A5)

**Comment by ghost** [comment]
I think the rainbow gradient on this [Dribbble post](https://dribbble.com/shots/1416538-Instagram-Handbook-for-Brands) might be worth taking a look at.

**Comment by theodorosploumis** [comment]
Hi, I could help with this if you like. I am not a logo designer but I like creating logos and graphics for open source projects. Should I proceed?

Thanks.

**Comment by dy** [comment]
Sure @theodorosploumis if you have ideas and motivation. I think this one is pretty decent though, so it is going to be not the worse. We can make votes.

**Comment by theodorosploumis** [comment]
You are right. A logo for a project with the term "Color" can lead to so many proposals and so many arguments that aren't suitable for an open source project. The logo above seems fine to me too.
